doc/rst/source/grdinterpolate.rst

```diff
@@ -1,5 +1,6 @@
 .. index:: ! grdinterpolate
 .. include:: module_core_purpose.rst_
+.. include:: /explain_array.rst_
```

Comment:
I have already included this file after `include:: explain_help.rst_` in my previous PR.